test/performance/workloads/performance-test-config.yaml

```diff
@@ -41,6 +41,12 @@ tuningSets:
   parallelismLimitedLoad:
     parallelismLimit: 100
 
+# PoissonLoad defines a load with with Poisson arrival time
```

Comment:
Arrival time of what ?  Maybe add ?

Reply:
The Poisson arrival time is for the load itself.